.github/workflows/dotnet-runtime-60.yaml

```diff
@@ -0,0 +1,52 @@
+name: dotnet-runtime-60
+
+on:
+  push:
+    branches: main
+    paths:
+      - dotnet-runtime-60/**
+  pull_request:
+    branches: main
+
+env:
+  SNAP_NAME: dotnet-runtime-60
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
question: Is my understanding correct that we do not build the runtime snaps for arm64, because these are the already existing snaps from Microsoft which never build for arm64?

Reply:
No, the fact that the current snaps never published arm64 (or did a long time ago) doesn't block us from doing so. I'd like to treat the "adding of arm64 runtime content snaps" as a separate feature, which will probably also switch Microsoft binaries to our binaries.